### DISCUSSION TITLE
Is there any way to set it to recognize other markers to finished tasks?

### DISCUSSION THREAD

**Discussion by Archie-2021** [tbabej/taskwiki#454] [Q&A]
Hey,
I want to mark some tasks in Obsidian mobile GUI/ preview mode , and obsidian by default uses `- [x]` instead of `- [X]` to mark finished tasks. This means that I have to manually mark them and that messes with my workflow. 

Is there anyway to change the config?

**Comment by liskin** [comment]
I don't think this is configurable: https://github.com/tools-life/taskwiki/blob/0ee63dd1f5d80be3296f14070211f4ade96d0e36/taskwiki/vwtask.py#L112-L116

**Reply by Archie-2021** [reply]
> I don't think this is configurable:
> 
> https://github.com/tools-life/taskwiki/blob/0ee63dd1f5d80be3296f14070211f4ade96d0e36/taskwiki/vwtask.py#L112-L116

Oh, so it is hardcoded..
thanks , i think I should create a feature request for it then?
